sphinx_sqlalchemy/main.py

```diff
@@ -97,8 +98,13 @@ def add_content(self, mapper: Mapper, definition: nodes.definition) -> None:
 
         # class documentation
         if mapper.class_.__doc__:
+            # Extract docstring and dedent to support both D212 and D213 multiline docstrings
+            docstring: str = mapper.class_.__doc__
+            docstring_lines = docstring.splitlines(keepends=True)
+            docstring_lines = [textwrap.dedent(line) for line in docstring_lines]
```

Comment:
```suggestion
            docstring_lines = prepare_docstring(mapper.class_.__doc__, self.state.document.settings.tab_width)
```

I think method from sphinx itself should be better 😄  https://github.com/sphinx-doc/sphinx/blob/fcc38997f1d9b728bb4ffc64fc362c7763a4ee25/sphinx/util/docstrings.py#L42

(your approach is not correct. since it would dedent lines in the body that should be indented)